packages/react-badge/src/components/PresenceBadge/usePresenceBadge.tsx

```diff
@@ -37,6 +37,16 @@ const iconMap = (
   }
 };
 
+export const DEFAULT_STRINGS = {
```

Comment:
Probably don't want to export these right now

```suggestion
const DEFAULT_STRINGS = {
```